packages/core-contracts/contracts/mocks/utils/SafeCastMock.sol

```diff
@@ -0,0 +1,40 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../../utils/SafeCast.sol";
+
+contract SafeCastMock {
```

Comment:
this mock could possibly be replaced with generated code by the new generator (it would be a storage library with an empty Data structure)

Reply:
Sure. Do you want to do something in particular in this PR, or are you just commenting for later improvements?